neqo-crypto/src/aead_fuzzing.rs

```diff
@@ -0,0 +1,73 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use crate::constants::{Cipher, Version};
+use crate::err::{sec::SEC_ERROR_BAD_DATA, Error, Res};
+use crate::p11::SymKey;
+use std::fmt;
+
+const FIXED_TAG: &[u8] = &[0x0; 16];
```

Comment:
```suggestion
const FIXED_TAG: &[u8] = &[0x0a; 16];
```

Let's make it non-zero.